NickvisionTagger.GNOME/Views/LyricsDialog.cs

```diff
@@ -238,18 +238,18 @@ private bool OnClose(Gtk.Widget sender, EventArgs e)
     /// <param name="e">EventArgs</param>
     private void AddSyncLyric(Gtk.Button sender, EventArgs e)
     {
-        var entryDialog = new EntryDialog(this, _iconName, _("New Synchronized Lyric"), "", _("Timestamp (hh:mm:ss)"), _("Cancel"), _("Add"))
+        var entryDialog = new EntryDialog(this, _iconName, _("New Synchronized Lyric"), "", _("Timestamp (hh:mm:ss, mm:ss.xx)"), _("Cancel"), _("Add"))
```

Comment:
```suggestion
        var entryDialog = new EntryDialog(this, _iconName, _("New Synchronized Lyric"), "", _("Timestamp (hh:mm:ss or mm:ss.xx)"), _("Cancel"), _("Add"))
```

Reply:
I understand but I guess some people may be unfamilliar with writing timestamp and will think that `hh:mm:ss, mm:ss.xx` is somehow one format, not 2 examples.

Reply:
> I understand but I guess some people may be unfamilliar with writing timestamp and will think that `hh:mm:ss, mm:ss.xx` is somehow one format, not 2 examples.

Yeah true